### DISCUSSION TITLE
Seb server Exam monitoring

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-server#124] [Q&A]
Hello. I would like to know where can I find a list that describes the function of the different indicators that can be monitored. I am interested in highlighting only those events linked to fraud, I am not interested in the battery for example. The most important thing is to know if the user is connecting from two different computers. Thank you so much!

**Comment by anhefti** [comment]
Hi,

Our case with two different computers and two logins within the same account on a supported assessment tool would be indicated in the monitoring by two different active connection with the same login/session name. These would be marked red (user session column) in the monitoring table and you would be able to see different IP Addresses for each connection. So you can either lock one or both connections with SEB Lockscreen function within the monitoring.

T check if a student uses a trusted SEB client, we recommend to ASK (App-Signature-Key) check where SEB clients needs to send a protected hash-key to SEB Server that gets verified on SEB Server. A trusted SEB client will generate the right key for the running Exam. On SEB SEB Server you are able to either grant trusted SEB client manually or choose a heuristic threshold of trust so SEB clients with same key gets trusted automatically if there are more of the same as the threshold.  

Please read more about this features in the [SEB Server User Documentation](https://seb-server.readthedocs.io/en/latest/exams.html ): 

Best regards

**Reply by Rapacioli** [reply]
Thank you very much!
